src/main/java/com/redhat/exhort/integration/providers/ProviderResponseHandler.java

```diff
@@ -19,12 +19,7 @@
 package com.redhat.exhort.integration.providers;
 
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Comparator;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Try to avoid these type of changes. Configure it in your IDE. It's ok for now but it adds too much noise in the PR